logspout/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
This file seems to be unused.

Reply:
Used by Logspout build process, added info to readme